agent/util-scripts/require-rpm

```diff
@@ -4,7 +4,7 @@
 # require-rpm - check to see if a RPM with the required version is installed.
 #
 # Returns 0 (success) if it is; 1 if it isn't; 2 if an error occured during
-# operation.
+# operation; 3 if the 3rd argument, match, is not recognized.
```

Comment:
Is it really useful to have a fourth error status?  I would lump an unrecognized match value in the same bucket as a missing parameter, etc. (`2`).

Reply:
Also, it would be helpful to the reader if this comment pointed out that "normal operation" included printing the EVR string to `stdout`.

Reply:
Well it was for me as I was writing the more detailed unit tests.  =)